### ISSUE TITLE
Wrong script loaded

### ISSUE THREAD

**Issue by aalbacetef** [elCarrera/snarqe#2]
https://github.com/elCarrera/snarqe/blob/307134d7f045b663a149fd26f5ec7a1a1c1e5007/index.html#L16

I think this might be wrong, script.js is missing.

**Comment by elCarrera** [comment]
Yes, I was just copy-pasting from glitch.me to here missing some details in the way. :P
